testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/asynch/AsyncPostProcessingTest.java

```diff
@@ -44,8 +47,16 @@
         war.addAsWebInfResource(AsyncPostProcessingTest.class.getPackage(), "AsyncPostProcessingTestWeb.xml", "web.xml");
         // Arquillian in the deployment
         war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(new ReflectPermission("suppressAccessChecks"),
+                new LoggingPermission("control", ""),
+                new PropertyPermission("arquillian.*", "read"),
+                new PropertyPermission("ipv6", "read"),
+                new PropertyPermission("node", "read"),
+                new PropertyPermission("org.jboss.resteasy.port", "read"),
                 new RuntimePermission("accessDeclaredMembers"),
-                new PropertyPermission("arquillian.*", "read")), "permissions.xml");
+                new RuntimePermission("getenv.RESTEASY_PORT"),
+                new SecurityPermission("insertProvider"),
```

Comment:
At some point during `client = new ResteasyClientBuilder().build();` it tries to initialise `PKCS7SignatureTextWriter` which in turn calls `BouncyIntegration.init()`, https://github.com/iweiss/Resteasy/blob/beddb34725eeb0371aca4d302f4e626a39bcd432/security/resteasy-crypto/src/main/java/org/jboss/resteasy/security/BouncyIntegration.java#L14-L14, which tries to add the BouncyCastleProvider/